helper/reports.go

```diff
@@ -55,3 +57,37 @@ func ProcessListReports(fileID int) error {
 	return nil
 
 }
+
+func ProcessDownloadReportCSV(reportID int, outputDirPath string) {
+	id := strconv.Itoa(reportID)
+	fmt.Println("Fetching Summary CSV Report for ID " + id + " ...")
```

Comment:
Check for verbose option and only do extra printing if versbose switch is on.

Reply:
removed all print statements for now. Will come back to verbose option later on